src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/integration/health/InfoTest.kt

```diff
@@ -8,17 +8,6 @@ import java.time.format.DateTimeFormatter
 
 class InfoTest : IntegrationTestBase() {
 
-  @Test
-  fun `Info page is accessible`() {
-    webTestClient.get()
-      .uri("/info")
-      .exchange()
-      .expectStatus()
-      .isOk
-      .expectBody()
-      .jsonPath("app.name").isEqualTo("Nomis User Roles Api")
```

Comment:
This is a really strange one - I had the same issue locally and couldn't figure out why the `app` section isn't visible when running the integration test hence removed this test. The test below still validates that the `/info` endpoint is visible

Investigated this by
- Running the test in debug mode and checked the property `info.app.name` in the Spring context. It's present and so `application.yml` is being loaded
![app-name-property](https://user-images.githubusercontent.com/2437376/143266397-aaa65045-29eb-46fb-8fec-61155ee2cc22.png)
- Running the test in debug mode and hit it the `/info` endpoint - the whole `app` section isn't there
![info-page-during-test](https://user-images.githubusercontent.com/2437376/143266526-7d177d1a-8c80-4562-93aa-5a06c472dd9e.png)

To make sure nothing has regressed I ran the app with `dev` profile and and checked `app` section is still visible
![info-page-with-dev-profile](https://user-images.githubusercontent.com/2437376/143266555-6d937644-b71a-4737-95b5-b6c889dd3c05.png)